pallets/valve/src/mock.rs

```diff
@@ -128,9 +128,12 @@ impl ExtBuilder {
 }
 
 pub(crate) fn events() -> Vec<pallet_valve::Event> {
-	System::events()
+	let evt = System::events()
```

Comment:
this normal to shorten in rust?

Reply:
its something I only see in mock files, but I see it a lot there